tests/fixtures/odl.py

```diff
@@ -260,3 +263,47 @@ def odl_api_test_fixture(odl_test_fixture: ODLTestFixture) -> ODLAPITestFixture:
     return ODLAPITestFixture(
         odl_test_fixture, library, collection, work, license, api, patron, client
     )
+
+
+class ODL2TestFixture(ODLTestFixture):
```

Comment:
@io7m I have used inheritance here for the ODL2 fixtures because they ODL2 API has almost no deviations from the ODL1 API.
This PR introduces the only changes between the 2 versions with respect to API actions.

Reply:
Normally I wouldn't like this, but there is an inheritance relationship between ODLAPI and ODL2API, so it makes sense that the fixtures might end up having one as well.